mobile/src/bundles/common/components/input/styles.ts

```diff
@@ -3,7 +3,32 @@ import { StyleSheet } from 'react-native';
 const styles = StyleSheet.create({
     input: {
         borderWidth: 1,
-        borderColor: 'gray',
+        borderColor: '#7e7e7e',
+        fontSize: 15,
+    },
+    label: {
+        color: '#000',
+        fontSize: 15,
+    },
+    error: {
+        borderWidth: 1,
+        borderColor: '#ff3d3d',
```

Comment:
Let's use Color enum everywhere where it's possible

Reply:
No, like this:  `borderColor: Color.SOME_COLOR,`

Reply:
Inside styles.ts

Reply:
Ok, i will change it now